package.json

```diff
@@ -30,7 +30,8 @@
     "lint"
   ],
   "dependencies": {
-    "openzeppelin-contracts-4.4.1": "https://github.com/OpenZeppelin/openzeppelin-contracts.git#4.4.1"
+    "openzeppelin-contracts-4.4.1": "https://github.com/OpenZeppelin/openzeppelin-contracts.git#4.4.1",
+    "openzeppelin-solidity": "2.0.0"
```

Comment:
OpenZeppelin has retired the package openzeppelin-solidity for openzepelin-contracts. The other code in this repo already uses the newer package.

I bet we can drop-in replace the the import statements in the oracle code with the new package reference. This will keep our dependencies cleaner.

The one wrinkle might be the solidity version in the oracle. It could be that the new package isn't compatible with the older solidity compiler version.

Could you give it a shot?

Reply:
Agree this dirties the dependencies. Unfortunately, the versions of solidity are not compatible. 

Our other option is to additionally store the 0.4.24 (or compatible) helper contracts in the this repo, thoughts?

Reply:
I'm good with it is, then. Thanks for checking!